src/stories/foundations/Typographyrules.stories.mdx

```diff
@@ -1,14 +1,14 @@
 import { Meta } from '@storybook/addon-docs'
-import Code from './assets/code-brackets.svg'
-import Colors from './assets/colors.svg'
-import Comments from './assets/comments.svg'
-import Direction from './assets/direction.svg'
-import Flow from './assets/flow.svg'
-import Plugin from './assets/plugin.svg'
-import Repo from './assets/repo.svg'
-import StackAlt from './assets/stackalt.svg'
-
-<Meta title="typography rules" />
+import Code from '../assets/code-brackets.svg'
+import Colors from '../assets/colors.svg'
+import Comments from '../assets/comments.svg'
+import Direction from '../assets/direction.svg'
+import Flow from '../assets/flow.svg'
+import Plugin from '../assets/plugin.svg'
+import Repo from '../assets/repo.svg'
+import StackAlt from '../assets/stackalt.svg'
```

Comment:
I [removed these imports](https://github.com/betterplace/betterplace-design-system/commit/37f63acf1b7aadf75f677a231ef1b06d38494620), because we don't need it. Ines also already changed the directory structure a bit, so this is probably going to cause a conflict.

Reply:
Yeah, I just moved it over, that's why we should merge it pretty soon, to clarify what's were, I'll probably just copy things over from you on merge